Sources/Supporting Files/Base.lproj/LaunchScreen.storyboard

```diff
@@ -11,15 +13,30 @@
             <objects>
                 <viewController id="01J-lp-oVM" sceneMemberID="viewController">
                     <view key="view" contentMode="scaleToFill" id="Ze5-6b-2t3">
-                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <rect key="frame" x="0.0" y="0.0" width="393" height="852"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
-                        <color key="backgroundColor" xcode11CocoaTouchSystemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                        <subviews>
+                            <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="logo" translatesAutoresizingMaskIntoConstraints="NO" id="T97-nu-3o4">
+                                <rect key="frame" x="142.66666666666666" y="373.66666666666669" width="108" height="105"/>
+                                <color key="backgroundColor" red="0.24889767169999999" green="0.46515023709999997" blue="0.75415229800000005" alpha="1" colorSpace="custom" customColorSpace="displayP3"/>
+                                <color key="tintColor" red="0.2470588088" green="0.46274507050000002" blue="0.75686275960000005" alpha="1" colorSpace="custom" customColorSpace="displayP3"/>
+                            </imageView>
+                        </subviews>
                         <viewLayoutGuide key="safeArea" id="6Tk-OE-BBY"/>
+                        <color key="backgroundColor" red="0.24889767169999999" green="0.46515023709999997" blue="0.75415229800000005" alpha="1" colorSpace="custom" customColorSpace="displayP3"/>
+                        <color key="tintColor" red="0.2470588088" green="0.46274507050000002" blue="0.75686275960000005" alpha="1" colorSpace="custom" customColorSpace="displayP3"/>
+                        <constraints>
+                            <constraint firstItem="T97-nu-3o4" firstAttribute="centerY" secondItem="Ze5-6b-2t3" secondAttribute="centerY" id="B9X-Zz-FXp"/>
+                            <constraint firstItem="T97-nu-3o4" firstAttribute="centerX" secondItem="Ze5-6b-2t3" secondAttribute="centerX" id="RTD-ZV-qjZ"/>
+                        </constraints>
                     </view>
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="iYj-Kq-Ea1" userLabel="First Responder" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="53" y="375"/>
+            <point key="canvasLocation" x="52.671755725190835" y="374.64788732394368"/>
```

Comment:
is this supposed to be different?

Reply:
It's not something we directly changed. All we've done is tell the auto layout to centre the image so I don't know why these values would have changed. Are they meant to be set at 53 and 375 do you know?